### PR TITLE
fixes Salicyclic Acid's overdose, updates the description of Salicyclic Acid and Oxandrolone

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Salicyclic Acid is supposed to double its brute healing when overdosing. However, due to an oversight, it instead healed burn too.
Also both Salocyclic Acid and Oxandrolone still had old descriptions from before they were changed, so I updated those too.

## Changelog
:cl:
add: Added brute healing to Salicyclic Acid's overdose.
add: More accurate descriptions to Salicyclic Acid and Oxandrolone.
del: Removed burn healing from Salicyclic Acid's overdose
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm
Comment (Sarchutar) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: shouldn't it actually deal brute on overdose instead of healing it since the description also says `Overdose will worsen existing bruising`?

Comment (EvilDragonfiend) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: `Overdose will worsen existing bruising` source?

Comment (Sarchutar) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]:
```
/datum/reagent/medicine/sal_acid
	name = "Salicyclic Acid"
	description = "Stimulates the healing of severe bruises. Extremely rapidly heals severe bruising and slowly heals minor ones. Overdose will worsen existing bruising."
```

Comment (CydiaLamiales) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Huh, you're right... However, Oxandrolone which is supposed to also worsen existing burns instead gets its burn healing doubled. And the wiki mentions that both Oxandrolone and Saliclyc Acid double their healing instead of dealing more damage when overdosing.

Comment (Sarchutar) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Okay then. Should probably change both their descriptions then, since Oxandrolone ones also says it worsens it.

Comment (CydiaLamiales) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Now I'm confused. Checked tg's wiki and code, both Saliclyc Acid and Oxandrolone have them actually deal damage instead of healing more when overdosing. What should it be then?

Comment (Sarchutar) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: I mean we did have a rebalance pr a long while ago which changed the code for it. #2995
It doesn't mention the double healing for it though, so i guess it could have been an oversight.
The thing it does mention about overdose is `balance: The overdose of most chems now causes organ damage instead of just reversing the effects of what the chem healed.`

I guess would probably be best to wait for a maintainers decision on that.

Comment (CydiaLamiales) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Yeah, seems like waiting for the higherups to make a decision would be the best option.

Comment (St0rmC4st3r) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Healing getting better on OD is intentional. The liver damage is enough of a downside for that.

Comment (CydiaLamiales) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:601]: Guess I'll just update the descriptions then.

**Review by itsmeow — APPROVED** [review]

**Review by ike709 — APPROVED** [review]

**Pull request merged**
